Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -125,6 +125,9 @@ public MobiledgeXIntegration(CarrierInfo carrierInfo = null, NetInterface netInt
 
             melMessaging = new MelMessaging(appName);
             matchingEngine.SetMelMessaging(melMessaging);
+#if UNITY_EDITOR
+            matchingEngine.EnableEdgeEvents = false;
```

Comment:
I should have remembered to communicate this earlier (especially since this flag was for the editor...).